languages/julia/exercises/concept/boolean-logic/.meta/config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "authors": [
+        {
+            "github_username": "SaschaMann",
+            "exercism_username": "SaschaMann"
+        }
+    ],
+    "contributors": [
+        {
+            "github_username": "cmcaine",
+            "exercism_username": "cmcaine"
+        }
+    ],
+    "language_versions": "≥1.0",
```

Comment:
Is unicode >= allowed?

Reply:
I've used it in all exercises so far. It's a free-form string so I doubt it'll cause issues.

Reply:
This is totally fine, as we're not doing anything with the field yet. Once we'll do something with the field, we'll revisit this and see if we want to change it.